pkg/migration/migrationmanager/manager.go

```diff
@@ -256,6 +256,10 @@ func (m *Manager) runMigration(
 			Settings: m.settings,
 		})
 	}
+	_, isSystemMigration := mig.(*migration.SystemMigration)
```

Comment:
I'm not looking at the right diff to be able to tell by myself, so just to be sure: make sure that SystemMigration and TenantMigration have comments that are good enough to explain to $average_crdb_developer what kind of migration they need, perhaps with a few examples.